README.md

```diff
@@ -0,0 +1,11 @@
+# WordPress VIP Node.js Skeleton
+
+Welcome to WordPress VIP! This repository is a starting point for building a Node.js application on WordPress VIP. It implements the basic requirements needed to successfully deploy and run an application on the platform. The entrypoint [`/src/index.js`][./src/index.js] is heavily annotated with comments that explain these requirements.
+
+This repository also includes [code examples](./examples) that demonstrate how to use various frameworks and technologies on the VIP Platform.
+
+## Documentation
+
+Further information, requirements, and best-practice advice can be found in WordPress VIP's documentation on Node.js:
+
+https://docs.wpvip.com/technical-references/node-js/
```

Comment:
Could this one be linked  in the above line "in WordPress VIP's documentation on Node.js"? Would look cleaner imo.